docs/FAQ/gui-faq-da.md

```diff
@@ -0,0 +1,151 @@
+Zonemaster
+==========
+
+1. [Hvad er Zonemaster?](#q1)
+2. [Hvem står bag Zonemaster?](#q2)
+3. [Hvordan kan Zonemaster hjælpe mig?](#q3)
+4. [Zonemaster rapporterer advarsler/fejl på mit domænenavn, hvad betyder det?](#q4)
+5. [Hvordan kan Zonemaster afgøre hvad der er rigtigt og forkert?](#q5)
+6. [Understøtter Zonemaster IPv6?](#q6)
+7. [Understøtter Zonemaster DNSSEC?](#q7) 
+8. [Hvad gør Zonemaster forkellig fra andre tilsvarende test-værktøjer?](#q8)
+9. [Zonemaster og privatliv](#q9)
+10. [Hvorfor kan jeg ikke teste mit domænenavn?](#q10)
+11. [Hvilke typer af forespørgsler genererer Zonemaster?](#q11)
+12. [Hvad er en "ikke-delegeret" test?](#undelegated)
+13. [Hvordan tester jeg en "reverse" zone med Zonemaster?](#q13)
+
+Zonemaster
+----------
+#### 1. Hvad er Zonemaster? <a name="q1"></a>
```

Comment:
This headline is not correctly rendered in Github. The problem could be that the space character is not U+0020 (normal space).

Headline "2" has the same problem.